parsers/NA.py

```diff
@@ -52,6 +50,31 @@ def get_text_from_image(link, expected_size, new_size, logger, session=None):
     return text
 
 
+def timestamp_checker(session=None, logger=None):
+
+    scada_info = get_text_from_image(session=session, link=timestamp_link, \
+                                    expected_size=(600,20), new_size=(1200,40), \
```

Comment:
it's a detail, but `\` is not necessary when you are inside any type of brackets. Your indentation is off by 1 char.

Reply:
I will fix that.